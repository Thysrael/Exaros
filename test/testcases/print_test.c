#include "stdio.h"

int main()
{
    printf("### print_test BEGIN ###\n");
    printf("hello world!\n");
    printf("### print_test END ###\n");
    return 0;
}