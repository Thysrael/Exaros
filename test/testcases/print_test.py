import sys

input = sys.stdin.readlines()[1:-1]
if len(input) != 1 or input[0].strip() != "hello world!":
    print("error")
