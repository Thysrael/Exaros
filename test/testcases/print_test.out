### TEST PRINT BEGIN ###
hello world!
### TEST PRINT END ###